homework/max-of-vector/maxOfVector.hpp

```diff
@@ -1,8 +1,25 @@
 #pragma once
 #include <vector>
 #include <limits>
+#include <algorithm>
+
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    int max = vec.at(0);
+    for_each(vec.begin(), vec.end(), [&max](int element){
+        if (element > max) {
+            max = element;
+        }
+    });
```

Comment:
If you want to use algorithm function, please use dedicated one. Here you should use std::max_element.